CMakeLists.txt

```diff
@@ -144,7 +144,14 @@ set(JPEGDIR "" CACHE PATH "GDL: Specify the JPEG directory tree")
 set(SZIPDIR "" CACHE PATH "GDL: Specify the SZip directory tree")
 
 set(GDL_DATA_DIR "/share/gnudatalanguage" CACHE PATH "GDL: data directory relative to CMAKE_INSTALL_PREFIX")
-
+set(GDL_LIB_DIR "" CACHE PATH "GDL: library directory relative to CMAKE_INSTALL_PREFIX")
+#define (for plplotdriver/CMakeLists.txt) the GDL_DRV_DIR where the drivers will be installed.
+#if GDL_LIB_DIR is empty, it will be the default (GDL_DATA_DIR/drivers) otherwise it is GDL_LIB_DIR (not GDL_LIB_DIR/drivers)
+if ( GDL_LIB_DIR STREQUAL "" OR NOT GDL_LIB_DIR)
+ set (GDL_DRV_DIR "${CMAKE_INSTALL_PREFIX}/${GDL_DATA_DIR}/drivers") # CACHE PATH "GDL: where the drivers will be installed.")
+else()
+ set (GDL_DRV_DIR "${GDL_LIB_DIR}" ) # CACHE PATH "GDL: where the drivers will be installed.")
```

Comment:
I guess I would have expected GDL_DRV_DIR to be "${GDL_LIB_DIR}/drivers", but I don't think this is particularly important.

Reply:
Well, I followed [my proposal](https://github.com/gnudatalanguage/gdl/pull/1481#issuecomment-1398638282) .
If I understand well, the object was to insure .so files would go to a /lib/ subdirectory, not necessarily something tagged 'gnudatalanguage/drivers', thats one folder too much IMHO.
With this PR, you can as well put everything in /gnudatalanguage or /gdl or...